sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryOptionsExtensions.kt

```diff
@@ -19,7 +19,9 @@ internal fun CocoaSentryOptions.applyCocoaBaseOptions(options: SentryOptions) {
     this.dsn = options.dsn
     this.attachStacktrace = options.attachStackTrace
     this.dist = options.dist
-    this.environment = options.environment
+    options.environment?.let {
+        this.environment = it
+    }
```

Comment:
Seems like the old way achieves the same thing.

Reply:
this.environment changed from nullable to requiring a non-null value